### PR TITLE
Add samwell

### PR DESCRIPTION
**Pull request by nh13**
Add [samwell](https://github.com/myriad-opensource/samwell)
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* [x] PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by nh13** [comment]
Initial commit added with `grayskull pypi samwell`

**Comment by nh13** [comment]
Relies on https://github.com/conda-forge/staged-recipes/pull/12421

File: recipes/samwell/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - cython >=0.29.15
```
Comment (nh13) [inline recipes/samwell/meta.yaml:24]:
```suggestion
    - cython >=0.29.15
    - poetry
```

File: recipes/samwell/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+about:
+  home: https://pypi.org/project/samwell/
+  summary: Useful utilities for biological data formats and analyses
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  skip-lints:
+    - cython-must-be-in-host
```
Comment (nh13) [inline recipes/samwell/meta.yaml:57]:
```suggestion
    - cython_must_be_in_host
```

**Comment by nh13** [comment]
@BiocondaBot please add label

**Comment by nh13** [comment]
@BiocondaBot please update

**Review by dpryan79 — APPROVED** [review]

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**